ereport/types/src/lib.rs

```diff
@@ -59,6 +61,59 @@ impl fmt::LowerHex for Ena {
     }
 }
 
+impl FromStr for Ena {
+    type Err = std::num::ParseIntError;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
```

Comment:
Nit: this could probably use a test or two, if only to help show the hex prefix usage